cli/install/install.go

```diff
@@ -0,0 +1,455 @@
+package install
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+
+	"github.com/apex/log"
+)
+
+// FlagInstall is a struct that contains all install flags.
+type FlagInstall struct {
+	InstallReinstall bool
+	InstallForce     bool
+	InstallVerbose   bool
+	InstallNoclean   bool
+}
+
+// Packet is a struct containing sys and install name of the packet.
+type Packet struct {
+	sysName     string
+	installName string
+}
+
+// isDeprecated checks wethere version of programm is below 1.10.0.
+func isDeprecated(version string) bool {
+	splitedVersion := strings.Split(version, ".")
+	if len(splitedVersion) < 2 {
+		return false
+	}
+	if splitedVersion[0] == "1" && len(splitedVersion[1]) < 2 {
+		return true
+	}
+	return false
+}
+
+// isCentOs checks if os is CentOS.
+func isCentOs() bool {
+	cmd := exec.Command("bash", "-c", "lsb_release -a")
+	readPipe, writePipe, _ := os.Pipe()
+	cmd.Stdout = writePipe
+	cmd.Start()
+	cmd.Wait()
+	writePipe.Close()
+	var buf bytes.Buffer
+	io.Copy(&buf, readPipe)
+	packets := buf.String()
+	return strings.Contains(packets, "CentOS")
+}
+
+// getLatestTarantoolTag returns latest tag from tarantool repository.
+func getLatestTarantoolTag() (string, error) {
+	cmd := exec.Command("git", "ls-remote", "--tags", "--sort="+
```

Comment:
Why not using `go-git` library?

Reply:
in case we won't be using go-git we should add the runtime dependency to the README (search and install commands)

Reply:
That library doesn't support ls-remote command, so not usable in our case.